### PR TITLE
(GH-513) Add keybinding for navigating sources

### LINKED ISSUE #513: Add keybinding for navigating between sources
We should pick some bindings for navigating quickly between sources views. Maybe some crtl+something+arrow keybind. Would also be nice to be able to Alt+Num or something to quickly navigate between the first 10 sources (as most people will have between 2-5 at most).

### PR DESCRIPTION
**Pull request by huffSamuel**
Fix 513 by adding keyboard bindings ctrl+1..9 to switch between respective source indexes.

Closes #513

### CODE REVIEW

**Comment by gep13** [comment]
@huffSamuel just wanted to say that I am not ignoring this PR, just been busy getting a few other things lined up, before I will be able to look at this properly.  Thanks for doing this!

File: Source/ChocolateyGui.Common.Windows/Views/ShellView.xaml [hunk 1]
```diff
@@ -22,6 +23,54 @@
...
+                <system:Int32>8</system:Int32>
+            </KeyBinding.CommandParameter>
+        </KeyBinding>
+        <KeyBinding Gesture="Ctrl+9" Command="{Binding GoToSourceCommand}">
+            <KeyBinding.CommandParameter>
+                <system:Int32>9</system:Int32>
+            </KeyBinding.CommandParameter>
+        </KeyBinding>
+
+    </Window.InputBindings>
```
Comment (ferventcoder) [inline Source/ChocolateyGui.Common.Windows/Views/ShellView.xaml]: Why 9 options to get to sources?

Comment (huffSamuel) [reply Source/ChocolateyGui.Common.Windows/Views/ShellView.xaml]: The intent was to support quick jumping to the first 9 sources in the list. I limited to 9 as that is a convenient number to stop at due to keyboard layout. Extending beyond that seemed somewhat excessive for quick access with a single key combination.

**Comment by gep13** [comment]
@huffSamuel your changes have been merged, thanks for your contribution 👍

**Pull request merged**